src/reinforcementAI.js

```diff
@@ -84,38 +84,42 @@ export default class ReinforcementAI {
     player0holdingCard1: reduxState.players[0].holdingCards[1],
   }
   */
-  allowedSAVectors(SObject) {
+  allowedSAVectors(SObject,reduxState) {
```

Comment:
Try not to pass the whole reduxState around, since it can be huge. Just pass in the necessary data.

Reply:
Should I include additional data in SObject?

Reply:
Yeah you should be able to do something like
```javascript
return {
  player0dead: reduxState.players[1].dead,
  player0lastCardId: player0lastCardId,
  player0holdingCard0: reduxState.players[1].holdingCards[0],
  player0holdingCard1: reduxState.players[1].holdingCards[1],
  // Add you new data here
  currentPlayerId: reduxState.currentPlayerId
}
```